src/index.js

```diff
@@ -7,7 +7,6 @@ export Tooltip from './components/containers/tooltip/Tooltip';
 export PopoverLink from './components/containers/popover/PopoverLink';
 export Modal from './components/containers/modal/Modal';
 
-export Label from './components/controls/Label';
```

Comment:
Any reason to remove this from the exports?

Reply:
Because of the dependency on theme now, it doesn't really work standalone. I wasn't sure how useful it was as an export (and we don't have an .md file for it so it's not documented), and it was so simple I didn't really want to clutter it with more logic. If you think it's useful I can rework it so it has a default theme and add some docs for it. Won't be difficult.